sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -708,9 +709,17 @@ public AnalyzedRelation visitValues(Values values, StatementAnalysisContext cont
                 columnValues
             ));
         }
-        FunctionIdent functionIdent = new FunctionIdent(ValuesFunction.NAME, Symbols.typeView(arrays));
-        FunctionImplementation implementation = functions.getQualified(functionIdent);
-        Function function = new Function(implementation.info(), arrays);
+
+        FunctionImplementation implementation = functions.resolveBuiltInFunctionBySignature(
```

Comment:
Could we use `functions.getQualified(ValuesFunction.SIGNATURE, Symbols.typeView(arrays))` instead? Looks like at this point all arguments are already validated and we could avoid signature matching...

Reply:
👍 sure, it makes more sense!